zerolog/README.md

```diff
@@ -0,0 +1,66 @@
+# Hertz zerolog
+This is a logger library that uses zerolog to implement the [Hertz logger interface](https://www.cloudwego.io/docs/hertz/tutorials/framework-exten/log/)
+
+## Usage
+
+Download and install it:
+
+    go get github.com/hertz-contrib/logger/zerolog
```

Comment:
code block will be better, user can copy directly
```bash
go get github.com/hertz-contrib/logger/zerolog
```

Reply:
Fixed